jupyter_client/kernelspec.py

```diff
@@ -138,17 +138,10 @@ def find_kernel_specs(self):
         return d
         # TODO: Caching?
 
-    def get_kernel_spec(self, kernel_name):
-        """Returns a :class:`KernelSpec` instance for the given kernel_name.
-
-        Raises :exc:`NoSuchKernel` if the given kernel name is not found.
+    def get_kernel_spec_helper(self, kernel_name, resource_dir):
```

Comment:
This is IMO misnamed, at least I have trouble seeing this NOT as a function, which gets a helper object, but it returns a kernel spec as far as I see it. So `_get_kernel_spec_for_name(name, dir)`?